### PR TITLE
Staging/ideaspace

### PR DESCRIPTION
**Pull request by dbradham**

### CODE REVIEW

**Review by RETprojects — APPROVED** [review]
The changes made look good to me!

**Comment by dbradham** [comment]
@dev-launchers/platform-enablement can you please take a look at these changes when you get a chance? CC: @Enjoy2Live

**Review by chungthuang — APPROVED** [review]

File: apps/ideaspace/src/components/common/CommonStyles.js
Comment (Enjoy2Live) [inline apps/ideaspace/src/components/common/CommonStyles.js:4]: it's better to have values coming from the theme rather than being fixed values

File: apps/ideaspace/src/components/common/CommonStyles.js
Comment (Enjoy2Live) [inline apps/ideaspace/src/components/common/CommonStyles.js:25]: it's better to have values coming from the theme rather than being fixed values.
especially typography.

Not a necessary change since eventually we'll be moving to tailwind

**Review by Enjoy2Live — APPROVED** [review]
you haven't changed alot of things in our code-spaces, I approve the changes but let's not merge it until we have master merged to release first.

**Pull request merged**
